src/pages/organizers/[organizerId]/preview/index.page.tsx

```diff
@@ -0,0 +1,267 @@
+                }}
+                size={ModalSizes.MD}
+              >
+                <Box padding={4}>
+                  <Trans
+                    i18nKey="organizers.ownerships.request.confirm_modal.body"
+                    values={{
+                      organizerName: organizerName,
+                    }}
+                  />
+                </Box>
+              </Modal>
+              {isOwnershipRequested && (
+                <Alert
+                  variant={AlertVariants.PRIMARY}
+                  marginBottom={4}
+                  fullWidth
+                >
+                  <Trans
+                    i18nKey="organizers.ownerships.request.pending"
+                    values={{
+                      organizerName: organizerName,
+                    }}
+                  />
+                </Alert>
+              )}
+              {isSuccessAlertVisible && (
+                <Alert
+                  variant={AlertVariants.SUCCESS}
+                  marginBottom={4}
+                  closable
+                  fullWidth
+                  onClose={() => {
+                    setIsSuccessAlertVisible(false);
+                  }}
+                >
+                  <Trans
+                    i18nKey="organizers.ownerships.request.success"
+                    values={{
+                      organizerName: organizerName,
+                    }}
+                  />
+                </Alert>
+              )}
+              {isErrorAlertVisible && (
+                <Alert
+                  variant={AlertVariants.DANGER}
+                  marginBottom={4}
+                  fullWidth
+                  closable
+                  onClose={() => {
+                    setIsErrorAlertVisible(false);
+                  }}
+                >
+                  <Trans i18nKey="organizers.ownerships.request.confirm_modal.error" />
+                </Alert>
+              )}
+
+              <Inline spacing={5}>
+                <Stack flex={3}>
+                  <OrganizerTable organizer={organizer} />
+                </Stack>
+                <Stack spacing={3.5} flex={1}>
+                  {!canEdit && isOwnershipEnabled && !isOwnershipRequested && (
+                    <Button
+                      variant={ButtonVariants.PRIMARY}
+                      onClick={() => setIsQuestionModalVisible(true)}
+                    >
+                      {t('organizers.detail.actions.request')}
+                    </Button>
+                  )}
+                  {canEdit && (
+                    <Button
+                      variant={ButtonVariants.SECONDARY}
+                      spacing={3}
+                      iconName={Icons.PENCIL}
+                      onClick={() =>
+                        router.push(`/organizer/${organizerId}/edit`)
+                      }
+                    >
+                      {t('organizers.detail.actions.edit')}
+                    </Button>
```

Comment:
Would use a `<Button variant={ButtonVariants.LINK}>` here instead for better accessibility

Reply:
I think Emma means a Link component with a button variant